src/character.h

```diff
@@ -402,14 +402,12 @@ class Character : public Creature, public visitable<Character>
         bool worn_with_flag( std::string flag ) const;
 
         // --------------- Skill Stuff ---------------
-        SkillLevel &skillLevel(const Skill* _skill);
-        SkillLevel &skillLevel(Skill const &_skill);
-        SkillLevel &skillLevel(const skill_id &ident);
+        SkillLevel &get_skill_level( const skill_id &ident );
```

Comment:
Do we need that thing? Having the ability to set skill level field value directly only prevents creation of reliable "on levelup" triggers for things like learning Brawling.